### PR TITLE
Doc file update

### PR DESCRIPTION
**Pull request by JerryZhangZZY**
Adding HowToJoin.md

### CODE REVIEW

**Comment by wangchao316** [comment]
this have english doc?

**Comment by JerryZhangZZY** [comment]
Not yet but I’ll work on it soon

从 Windows 版邮件发送

发件人: wangchao316
发送时间: 2022年2月10日 下午 12:09
收件人: apache/iotdb
抄送: Zhang Zeyu; Author
主题: Re: [apache/iotdb] Doc file update (PR #5027)

this have english doc?
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

File: docs/Development/HowToJoin.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# IoTDB R&D Beginning
```
Comment (HTHou) [inline docs/Development/HowToJoin.md:1]: Add Apache license header?

File: site/src/main/.vuepress/config.js [hunk 2]
```diff
@@ -116,6 +116,7 @@ var config = {
 					items: [
 					  { text: 'How to vote', link: '/Development/VoteRelease'},
 					  { text: 'How to Commit', link: '/Development/HowToCommit'},
+					  { text: 'Become a Contributor', link: '/zh/Development/HowToJoin'}
```
Comment (HTHou) [inline site/src/main/.vuepress/config.js:119]:
```suggestion
					  { text: 'Become a Contributor', link: '/Development/HowToJoin'}
```

**Pull request merged**